RecoLocalCalo/HGCalRecProducers/python/hgcalLayerClusters_cfi.py

```diff
@@ -2,19 +2,31 @@
 
 #### PF CLUSTER ECAL ####
 
+from RecoLocalCalo.HGCalRecProducers.HGCalRecHit_cfi import dEdX_weights, HGCalRecHit
+
+from RecoLocalCalo.HGCalRecProducers.HGCalUncalibRecHit_cfi import HGCalUncalibRecHit
+
+from SimCalorimetry.HGCalSimProducers.hgcalDigitizer_cfi import fC_per_ele, nonAgedNoises, hgchebackDigitizer
+
 #energy corrector for corrected cluster producer
 hgcalLayerClusters =  cms.EDProducer(
-    "HGCalClusterTestProducer",
+    "HGCalClusterProducer",
     detector = cms.string("all"),
     doSharing = cms.bool(False),
-    deltac = cms.double(2.),
-    ecut = cms.double(0.01),
-    kappa = cms.double(10.),
-    multiclusterRadius = cms.double(0.015),
+    deltac = cms.vdouble(2.,2.,2.),
+    dependSensor = cms.bool(True),
+    ecut = cms.double(3.),
+    kappa = cms.double(9.),
+    multiclusterRadii = cms.vdouble(2.,2.,2.),
```

Comment:
There is no `cms.vfloat`, so I'm not sure what you mean. I don't think we gain much by converting a three-element `vector<double>` to `vector<float>` either.